lib/AST/DiagnosticList.cpp

```diff
@@ -24,6 +24,11 @@ enum class swift::DiagID : uint32_t {
 static_assert(static_cast<uint32_t>(swift::DiagID::invalid_diagnostic) == 0,
               "0 is not the invalid diagnostic ID");
 
+enum class swift::FixItID : uint32_t {
+#define FIXIT(ID, Text, Signature) ID,
+#include "swift/AST/FixIts.def"
```

Comment:
Does it make sense to have fix-its in their own file? I would have thought it'd be simplest to have them next to the diagnostics.

Reply:
Defining fix-its alongside the other diags complicates the macro usage by a fair amount. I'm starting to think the advantages are outweigh the tradeoffs though, so updated the PR to do this. I only added the definitions to Parse and Sema diags for now, they can be rolled out to the other headers as needed.